pkg/disk/partition_table_test.go

```diff
@@ -1,26 +1,24 @@
-package disk
+package disk_test
 
 import (
 	"testing"
 
+	"github.com/osbuild/images/internal/testdisk"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestPartitionTableFeatures(t *testing.T) {
-	type testCase struct {
-		partitionType    string
-		expectedFeatures partitionTableFeatures
-	}
-	testCases := []testCase{
-		{"plain", partitionTableFeatures{XFS: true, FAT: true}},
-		{"luks", partitionTableFeatures{XFS: true, FAT: true, LUKS: true}},
-		{"luks+lvm", partitionTableFeatures{XFS: true, FAT: true, LUKS: true, LVM: true}},
-		{"btrfs", partitionTableFeatures{XFS: true, FAT: true, Btrfs: true}},
-	}
+func TestPartitionTable_GetMountpointSize(t *testing.T) {
+	pt := testdisk.MakeFakePartitionTable("/", "/app")
 
-	for _, tc := range testCases {
-		pt := testPartitionTables[tc.partitionType]
-		assert.Equal(t, tc.expectedFeatures, pt.features())
+	size, err := pt.GetMountpointSize("/")
+	assert.NoError(t, err)
+	assert.Equal(t, testdisk.FakePartitionSize, size)
 
-	}
+	size, err = pt.GetMountpointSize("/app")
+	assert.NoError(t, err)
+	assert.Equal(t, testdisk.FakePartitionSize, size)
+
+	// non-existing
+	_, err = pt.GetMountpointSize("/custom")
+	assert.Error(t, err)
```

Comment:
I wonder if it makes sense to test for (parts of?) the specific error, I find this usually a nice pattern to also check if the error message is good or if it's one of the: `cannot find foo: foo not found: when trying to read foo it was not found` kind of situation that is so easy to hit in go :)

Reply:
Added a check! :)